build_tools/cmake/rebuild.sh

```diff
@@ -35,10 +35,5 @@ else
   mkdir build
```

Comment:
Just commenting here so there's a thread.

> (see commit messages for details)

Please put the details in the PR description, since that's what's going to be the commit description when this is squash merged (assuming you're going to squash merge. I don't think the history is interesting here)

Reply:
I would be creating a merge commit. Otherwise I would split the first commit into its own PR. This was just easier.

Reply:
Ah, fair enough

Reply:
Although I'm still a proponent of aims for a nice experience with `git log --first-parent` which means making merge commits act just like regular commits with some bonus metadata (and therefore describe everything they do).

Reply:
Added more details to the PR description.